packages/react/src/primitives/Button/__tests__/Button.test.tsx

```diff
@@ -46,6 +46,239 @@ describe('Button test suite', () => {
     expect(menu.classList).toContain(`${ComponentClassNames['Button']}--menu`);
   });
 
+  it('should render color themes for default variation', async () => {
```

Comment:
To isolate test behavior can we break these out in to individual tests using `it.each`? Example:
```
const DEFAULT_SUPPORTED_COLOR_THEMES= ['info', 'success']


it.each(DEFAULT_SUPPORTED_COLOR_THEMES)('should render the %s color theme for the default variation', async (colorTheme) => {
  const testId = `default${capitalize(colorTheme)}ColorTheme`
  const button = await screen.findByTestId(testId);
  expect(button.classList).toContain(`amplify-button--outlined--${colorTheme}`);
})
```

Same question for 103, 173, and 245.

Reply:
@calebpollman Updated this in latest if you can please take a look

Reply:
@calebpollman Oops I missed the line 245 part, do you think we should have tests for each colorTheme for the unsupported variations?

Reply:
Forgive me if I'm totally off here, but those variations are to be removed in the future correct?

Reply:
No worries, that's right! We should deprecate warning and destructive variations.

The `menu` variation is a little weird, it's used specifically for our Menu primitive and we don't _reallllly_ advertise it as an option (like we don't include it in our Button demo on the docs, its buried down further on the page, and we don't support it in Figma). This is one of our Radix primitives, which we've considered refactoring to drop the dependency. I think for now it's safe to not have it support colorTheme and maybe in the future we can revisit depending on where we net out with the Menu primitive.

Reply:
Cool, since those tests aren't going to be around for long not too concerned